### ISSUE TITLE
continue training 

### ISSUE THREAD

**Issue by nrjvarshney** [PyTorchLightning/pytorch-lightning#3142]
#### What is your question?
I have 10 training datasets. I want to train a model sequentially on these datasets i.e train on first training dataset then train the same model on the second dataset and so on. 

How to do this?

   
```
model = ClassificationModel()
for dataset in training_datasets:
    trainer = pl.Trainer(gpus=-1)
    trainer.fit(model)
```

**Comment by Borda** [comment]
the simplest way would be merging these datasets in single datamodule and train it at once, another way is passing multiple dataloaders... the least way is just loading checkpoint model after each training and run the training with another dataset on row... feel free to reopen if needed 🐰

**Comment by nrjvarshney** [comment]
Do I need to load the checkpoint model after each training, if I'm initializing the model above the loop?

**Comment by rohitgr7** [comment]
nope, reload checkpoint is not required. Also, this will reset your optimizers and schedulers if you call `.fit` on each dataset. If you want that then it's ok, else you have to do what @Borda suggested.

**Comment by nrjvarshney** [comment]
@Borda , @rohitgr7 , It would be great if you could share code for this. I'm a bit confused about this. 
I intend to train a model sequentially on 10 datasets i.e train on dataset_1 then on dataset_2 and so on. Should the learning rate, optimizer be reset after training on dataset_1 and subsequently after every training?
Thanks.
